packages/fx-core/src/plugins/resource/aad/plugin.ts

```diff
@@ -569,7 +573,9 @@ export class AadAppForTeamsImpl {
     }
 
     if (botEndpoint) {
-      redirectUris.web?.redirectUris?.push(`${botEndpoint}/auth-end.html`);
+      redirectUris.web?.redirectUris?.push(
+        isVs ? `${botEndpoint}/bot-auth-end.html` : `${botEndpoint}/auth-end.html`
```

Comment:
Why use different redirect url in vscode and vs?

Reply:
According to Di's investigation, we need to separate tab and bot's auth pages in VS since we may host backend in the same webapp.